src/jit/bitsetasuint64inclass.h

```diff
@@ -295,16 +295,15 @@ class BitSetUint64Iter
     {
     }
 
-    bool NextElem(unsigned* pElem)
+    int NextElem()
     {
         static const unsigned UINT64_SIZE = 64;
 
         if ((m_bits & 0x1) != 0)
         {
-            *pElem = m_bitNum;
             m_bitNum++;
             m_bits >>= 1;
-            return true;
+            return m_bitNum++;
         }
```

Comment:
ok, never mind, since you need to return the before-incremented value.

But, you need to delete the `m_bitNum++` line above

---
In reply to: [122268396](https://github.com/dotnet/coreclr/pull/11945#discussion_r122268396) [](ancestors = 122268396)